pycheribuild/projects/project.py

```diff
@@ -1612,10 +1612,9 @@ def update(self, current_project: "Project", *, src_dir: Path, base_project_sour
 class DefaultInstallDir(Enum):
     DO_NOT_INSTALL = "Should not be installed"
     IN_BUILD_DIRECTORY = "$BUILD_DIR/test-install-prefix"
-    ROOTFS = "The rootfs for this target"
+    ROOTFS = "The rootfs for this target (<rootfs>/opt/<arch>/<program> by default)"
     COMPILER_RESOURCE_DIR = "The compiler resource directory"
-    SYSROOT = "The sysroot for this target"
-    SYSROOT_AND_ROOTFS = "The sysroot for this target and the sysroot"
+    SYSROOT = "The sysroot for this target (<rootfs>/usr/local/<arch> by default)"
```

Comment:
Maybe we should call these OPTBASE+LOCALBASE instead or similar? Having SYSROOT and ROOTFS both exist, with both going into the rootfs by default, just different places, is a bit confusing.

Reply:
(and I don't think there's a case when you ever want to install something to the sysroot but _not_ the corresponding directory in the rootfs when the two are different; you don't always _need_ to if you're statically linking/it's a header-only library and you don't want to support native compilation, but you still _should_)

Reply:
I also think this or a similar renaming is a good idea so that the `path_in_rootfs` attribute doesn't mistakenly get interpreted as relative to the `/opt/...` directory.